src/HealthGPS.Datastore/CMakeLists.txt

```diff
@@ -1,11 +1,21 @@
 find_package(fmt CONFIG REQUIRED)
 find_package(jsoncons CONFIG REQUIRED)
+find_package(libzippp CONFIG REQUIRED)
 
 add_library(HealthGPS.Datastore STATIC "")
 target_compile_features(HealthGPS.Datastore PUBLIC cxx_std_${CMAKE_CXX_STANDARD})
 
-target_sources(HealthGPS.Datastore PRIVATE "datamanager.cpp" "datamanager.h" "api.h")
+target_sources(
+    HealthGPS.Datastore
+    PRIVATE "api.h"
+            "datamanager.cpp"
+            "datamanager.h"
+            "schema.cpp"
+            "schema.h"
+            "zip_file.cpp"
+            "zip_file.h")
```

Comment:
For my own benefit could you give me a 1 sentence explanation of what this does?

Reply:
A short sentence, obviously :P

Reply:
It's just a list of source files for the `HealthGPS.Datastore` library. You have to be explicit with CMake/C++.

(Sorry that was two sentences :stuck_out_tongue:.)